modules/search/instant-search/components/gridicon/index.jsx

```diff
@@ -159,10 +165,11 @@ class Gridicon extends Component {
 			<svg
 				className={ iconClass }
 				height={ size }
-				width={ size }
 				onClick={ this.props.onClick }
+				style={ { height: size, width: size } }
+				viewBox={ '0 0 ' + size + ' ' + size }
```

Comment:
I'm sure this could be cleaner, but it was the only way I could think to make it work.

Reply:
👍 This looks good to me.